resources/gui/js/apis/sidebar.js

```diff
@@ -0,0 +1,194 @@
+   *
+   * @param pluginName {string}
+   * @param icon {string}
+   * @param content {string}
+   */
+  addPluginTab(pluginName, icon, content) {
+    const tab = CosmoScout.loadTemplateContent('sidebar-plugin-tab');
+    if (tab === false) {
+      return;
+    }
+
+    tab.innerHTML = this.replaceMarkers(tab.innerHTML, this._makeId(pluginName), icon, content);
+
+    this._sidebar.insertBefore(tab, this._sidebarTab);
+  }
+
+  /**
+   * Add a new section to the settings tab
+   *
+   * @param sectionName {string}
+   * @param icon {string}
+   * @param content {string}
+   */
+  addSettingsSection(sectionName, icon, content) {
+    const tab = CosmoScout.loadTemplateContent('sidebar-settings-section');
+    if (tab === false) {
+      return;
+    }
+
+    const html = this.replaceMarkers(tab.innerHTML, this._makeId(sectionName), icon, content);
+
+    tab.innerHTML = html
+      .replace(this.regex('SECTION'), sectionName)
+      .trim();
+
+    this._settings.appendChild(tab);
+  }
+
+  /**
+   * Passthrough call to CosmoScout.clearHtml
+   *
+   * @param id {string}
+   */
+  clearContainer(id) {
+    CosmoScout.clearHtml(id);
+  }
+
+  /**
+   * TODO UNUSED / Remove jQuery
+   *
+   * @param id {string}
+   */
+  clearDropdown(id) {
+    CosmoScout.clearHtml(id);
+
+    $(`#${id}`).selectpicker('render');
+  }
+
+  /**
+   * Sets an elevation data copyright tooltip
+   * TODO Remove jQuery
+   *
+   * @param copyright {string}
+   */
+  setElevationDataCopyright(copyright) {
+    $('#img-data-copyright').tooltip({ title: `© ${copyright}`, placement: 'top' });
+  }
+
+  /**
+   * Sets a map data copyright tooltip
+   * TODO Remove jQuery
+   *
+   * @param copyright {string}
+   */
+  setMapDataCopyright(copyright) {
+    $('#dem-data-copyright').tooltip({ title: `© ${copyright}`, placement: 'bottom' });
+  }
+
+  /**
+   * Adds an option to a dropdown
+   * TODO remove jQuery
+   *
+   * @param id {string} DropDown ID
+   * @param value {string|number} Option value
+   * @param text {string} Option text
+   * @param selected {boolean} Selected flag
+   */
+  addDropdownValue(id, value, text, selected = false) {
+    const dropdown = document.getElementById(id);
+    const option = document.createElement('option');
+
+    option.value = value;
+    option.selected = selected === true;
+    option.text = text;
+
+    if (dropdown !== null) {
+      dropdown.appendChild(option);
+
+      $(`#${id}`).selectpicker('refresh');
+    } else {
+      console.warn(`Dropdown '${id} 'not found`);
+    }
+  }
+
+  /**
+   * TODO UNUSED
+   *
+   * @param id {string}
+   * @param value {string|number}
+   */
+  setDropdownValue(id, value) {
+    $(`#${id}`).selectpicker('val', value);
+  }
+
+  /**
+   * TODO UNUSED
+   * @param id {string}
+   */
+  setRadioChecked(id) {
+    this.setCheckboxValue(id, true);
+  }
+
+  /**
+   * TODO UNUSED
+   * @param id {string}
+   * @param value {boolean}
+   */
+  setCheckboxValue(id, value) {
```

Comment:
This could be moved to the `CosmoScout` object. Somewhere close to `initSlider()`.